stubs/D3DShot/METADATA.toml

```diff
@@ -2,6 +2,6 @@ version = "0.1.*"
 requires = ["types-Pillow"]
 
 [tool.stubtest]
-# The library only works on Windows; we currently only run stubtest on Ubuntu for third-party stubs in CI.
-# See #8660
+# TODO: re-enable
```

Comment:
```suggestion
# TODO: re-enable
```

Reply:
Wait, this TODO comment I _did_ like -- it would be good if we could figure out how to enable stubtest for this package!